.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Python package build and publish
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+
+    - name: Install dependencies
+      run: |
+        python3 -m pip install --upgrade pip
+        pip3 install scikit-build twine
+
+    - name: Build Python sdist
+      run: python3 setup.py sdist
+
+    - name: Build manylinux Python wheels
+      uses: RalfG/python-wheels-manylinux-build@v0.2.2-manylinux2014_x86_64
+      with:
+        python-versions: 'cp36-cp36m cp37-cp37m cp38-cp38'
+        build-requirements: 'scikit-build'
+        system-packages: 'make cmake bzip2-devel expat-devel libxml2-devel xz-devel'
+
+    - name: Publish wheels to PyPI
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
Is it necessary to use the password here, or could an API token be used instead?

Reply:
It's a little awkward to do but it looks like it's possible